button-examples/Web/web-button-example.md

```diff
@@ -19,43 +21,52 @@ Before you can use a button, you will need to install the following:
 * [Add a theme](#add-a-theme)
 * [Import JavaScript effects](import-javascript-effects)
 
-### Install `mcd-button` or `mdc-icon-button`
+### Install
 Install the `mdc-button` component before including it in your source.
 
+TODO: Add subsection for CDN users (Who do not use any bundler).
+
 **`mdc-button`**
 ```bash
-npm install @material/buttons
+npm install @material/button
 ```
 
+If you would like to use Icons as buttons:
+
 **`mdc-icon-button`**
 ```bash
 npm install @material/icon-button
 ```
 
-### Add a theme
-Both `mdc-button` and `mdc-icon-button` work with themes (styles). Import a style into your `*.scss*` stylesheet to apply it to your website, including buttons or toggle icon buttons:
```

Comment:
We will use the canonical guidance terminology in this document rather than the terminology used in the API docs for consistency.

Reply:
`Add a theme` section feels like it is jumping right into theming without explaining how to use base styles first.

WDYT?